test/functional/bip68-sequence.py

```diff
@@ -371,8 +371,10 @@ def activateCSV(self):
         # activation should happen at block height 432 (3 periods)
         min_activation_height = 432
         height = self.nodes[0].getblockcount()
-        assert(height < min_activation_height)
```

Comment:
Nit, spaces around operators.

Reply:
also, prefer `assert_greater_than()` where possible (since it will print the result of both sides of the inequality if the assert fails)